tests/kuttl/test-suites/default/deps/infra.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+spec:
+  storageClass: crc-csi-hostpath-provisioner
+  mariadb:
+    enabled: false
```

Comment:
we shoudl double check if this has been removed by the way.
I'm not sure if we completed the cleanup of that in the CRDs yet.